drivers/platform/xilinx/xilinx_uart.c

```diff
@@ -37,7 +37,9 @@
 #include <stdlib.h>
 #include "no_os_error.h"
 #include "no_os_uart.h"
+#include "no_os_irq.h"
```

Comment:
i am surprised that by adding these files doesn't break existing projects, it seems all of them already include irq

Reply:
Sometimes life has positive surprises.